### PR TITLE
Angle is 4 Bytes

### PR DESCRIPTION
**Pull request by hobbyquaker**

### CODE REVIEW

File: notifications.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* 0f 00 04 32 01 17 00 00 00 00 10 00 00 00 10 - ?
+* 0f 00 04 3a 01 28 00 00 00 00 10 00 00 00 02 - ?
+* 0f 00 04 3b 01 15 00 02 00 00 00 02 00 00 00 - ?
+* 0f 00 04 3c 01 14 00 02 00 00 00 02 00 00 00 - ?
+
+* 05 00 82 37 01 - Motor A (0x37) starts movement
+* 05 00 82 37 0a - Motor A (0x37) stops movement
+* 05 00 82 37 05 - Motor A (0x37) seems to report obstruction/overload
+
+* 05 00 82 32 0a - seems to report a change of the LED state
```
Comment (helje5) [inline notifications.md]: Yes it is. Packet 0x82 is a port value change notification. `37` is the port of a motor, `32` is the port of the LED. `0a` would be white, was it white? ;-)

Comment (hobbyquaker) [reply notifications.md]: tried different colors, always received 0a

Comment (helje5) [reply notifications.md]: what about distance?

Comment (hobbyquaker) [reply notifications.md]:
```
> <Buffer 08 00 81 32 11 51 00 06>
< <Buffer 05 00 82 32 0a>
> <Buffer 08 00 81 32 11 51 00 03>
< <Buffer 05 00 82 32 0a>
> <Buffer 08 00 81 32 11 51 00 00>
< <Buffer 05 00 82 32 0a>
```

I think `0a` means something like "action completed".

Comment (helje5) [reply notifications.md]: Sounds reasonable, so 0x05 would be conflict and 0x01 started.

Comment (hobbyquaker) [reply notifications.md]: distance and color sensor are still a riddle for me. No reports received until now. I guess they have to be activated first?

Comment (helje5) [reply notifications.md]: Works for me, yes you need to activate notifications as described.

```
                 /- Port (color/distance sensor)
        /- len!  |
        |        |
        0. 1. 2. 3. 4. 5. 6. 7. 8. 9.
  data: 0a 00 41 01 08 01 00 00 00 01   - Port: color/distance sensor[0x25]
              |
              \- packet type, register for notifications
```

Comment (hobbyquaker) [reply notifications.md]: Ah thx! 👍

Comment (helje5) [reply notifications.md]: I wonder how you stop them (or whether you would want to, battery?) Maybe setting 6th or 9th to 0.

Comment (hobbyquaker) [reply notifications.md]: hmm... I fear I'm holding it wrong ;)
when sending `0a 00 41 25 08 01 00 00 00 01` the LED of the distance sensor starts to cycle through different colors, the led on the hub stays red, it doesn't send any notifications and doesn't react on commands. Have to remove batteries to get it alive again

Comment (hobbyquaker) [reply notifications.md]: ah - got it. as soon as you're doing it right things start working ;) `0a 00 41 01 08 01 00 00 00 01`

Comment (helje5) [reply notifications.md]: not 25, 01. 25 is the device type, 01 is the channel.

Comment (helje5) [reply notifications.md]: I sincerely hope you are not trying to do what I planned to do ;->

Comment (hobbyquaker) [reply notifications.md]: and yes - setting the last byte to `00` unsubscribes.

Comment (hobbyquaker) [reply notifications.md]: what do you plan to do? :D

**Pull request closed without merging**